### ISSUE TITLE
Refer a cell in the chat window

### ISSUE THREAD

**Issue by andypetrella** [andypetrella/spark-notebook#557]
using ALT+Click for instance

**Comment by andypetrella** [comment]
cc @vidma

**Comment by vidma** [comment]
I think ideally next to each cell there should be a comment button (contextual menu, or become visible when chat activated - of course this would ask for further rework/improvements to cell presentation).

**Comment by vidma** [comment]
in fact I think it would be nicer to simply write comments by each cell (not a chat window). like in google docs kinda.

**Comment by andypetrella** [comment]
yep this is another feature to me, I was thinking the same. but referencing from the chat would be interesting too,  I see a lot of use cases
